releases/release-1.11/release_notes_draft.md

```diff
@@ -95,16 +95,32 @@ This tag is for changes that don't affect the user experience, such as behind-th
 * Default mount propagation has changed from "HostToContainer" ("rslave" in Linux terminology), as it was in 1.10, to "None" ("private") to match the behavior in 1.9 and earlier releases. "HostToContainer" as a default caused regressions in some pods. ([#62462](https://github.com/kubernetes/kubernetes/pull/62462), [@jsafrane](https://github.com/jsafrane))
 * The kube-apiserver `--storage-version` flag has been removed; you must use `--storage-versions` instead. ([#61453](https://github.com/kubernetes/kubernetes/pull/61453), [@hzxuzhonghu](https://github.com/hzxuzhonghu))
 * [Pod priority and preemption](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/)
-is enabled by default. Even if you don't plan to use this feature,
-you might need to take some action right after upgrading.
-In multi-tenant clusters where not all users are trusted, you are advised to create appropriate
-amount of quota for two default priority classes, `system-cluster-critical` and
-`system-node-critical`. These two priority classes are added to clusters by default.
-[ResourceQuota](https://kubernetes.io/docs/concepts/policy/resource-quotas/)
-should be created to limit users from creating Pods at these priorities if not all users of your
-cluster are trusted. We do not advise disabling this feature since critical system Pods rely on
-the scheduler preemption to be scheduled when cluster is under resource pressure.
- 
+is enabled by default.\
+  Note that **it will be possible for users of the cluster to create pods that block some system daemons from running, and/or evict system daemons that are already running, by creating pods at the `system-cluster-critical` and `system-node-critical` priority classes, which are present in all clusters by default.** Please read the following information to understand the details. This is particularly important for those who have untrusted users in their Kubernetes clusters.
+
+  There are two kinds of critical system daemons in Kubernetes -- ones that run per-node as DaemonSets (e.g. fluentd, XXX list the rest of them here) and ones that run per-cluster (possibly more than one instance per cluster, but not one per node) (e.g. DNS, heapster, XXX list the rest of them here).
```

Comment:
Fix the first XXX to list the other node-level critical system pods and the second XXX to list the other cluster-level critical system pods.

Reply:
Done.